dnf/automatic/main.py

```diff
@@ -304,9 +314,10 @@ def main(args):
     (opts, parser) = parse_arguments(args)
 
     try:
+        conf = None
```

Comment:
I recommend defining these variables out of the try scope. Theoretically, if an exception happened before their definition, you would get the same problem you try to avoid with "in not None" tests down in the except block.

Reply:
Thanks, changed.